documentation.html

```diff
@@ -0,0 +1,1440 @@
+              <button class="btn btn-icon icon-text-down">
+                <i class="fas fa-heart icon-med icon-badge colorRed">
+                  <span class="badge circle bdg-tr bdg-num bdg-blue">1</span>
+                </i>
+                <p class="mt-05">Wishlist</p>
+              </button>
+            </div>
+          </nav>
+          <pre class="m-1-0"><code class="html border-round">
+&lt;nav class="nav top-fixed p-1 box-shadow-down"&gt;
+  &lt;h1 class="font-size-m"&gt;Brand Name&lt;/h1&gt;
+  &lt;div class="menu"&gt;
+    &lt;button class="btn btn-icon icon-text-down"&gt;
+      &lt;i class="fas fa-store icon-med"&gt;&lt;/i&gt;
+      &lt;p class="mt-05"&gt;Store&lt;/p&gt;
+    &lt;/button&gt;
+    &lt;button class="btn btn-icon icon-text-down"&gt;
+      &lt;i class="fas fa-shopping-cart icon-med icon-badge"&gt;
+        &lt;span class="badge circle bdg-tr bdg-num bdg-blue"&gt;4&lt;/span&gt;
+      &lt;/i&gt;
+      &lt;p class="mt-05"&gt;Cart&lt;/p&gt;
+    &lt;/button&gt;
+    &lt;button class="btn btn-icon icon-text-down"&gt;
+      &lt;i class="fas fa-heart icon-med icon-badge colorRed"&gt;
+        &lt;span class="badge circle bdg-tr bdg-num bdg-blue"&gt;1&lt;/span&gt;
+      &lt;/i&gt;
+      &lt;p class="mt-05"&gt;Wishlist&lt;/p&gt;
+    &lt;/button&gt;
+  &lt;/div&gt;
+&lt;/nav&gt;
+        </code></pre>
+          <h2 class="font-size-m mt-1">
+            - Left fixed Navbar (floats from right on mobile, see the nav of
+            this page for working eg.)
+          </h2>
+          <p class="medium font-size-sm mt-05">Code for nav (paste in body)</p>
+          <pre class="m-1-0"><code class="html border-round">
+&lt;nav class="leftfixed-nav p-1-0" id="menu"&gt;
+  &lt;div class="top-element"&gt;
+    &lt;a href="/" class="logo"&gt;
+      &lt;img src="/images/panda.png" alt="" /&gt;
+      &lt;h3&gt;Brandname&lt;/h3&gt;
+    &lt;/a&gt;
+    &lt;button class="m-0-05 btn btn-icon" id="menu-close"&gt;
+      &lt;i class="fas fa-times icon-med"&gt;&lt;/i&gt;
+    &lt;/button&gt;
+  &lt;/div&gt;
+  &lt;a class="nav-element-link mt-05 active" href="#"&gt;Installation&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Utilities&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Text Utilities&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Alerts&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Avatars&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Badges&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Buttons&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Cards&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Navigation&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Input&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Images&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Modal&lt;/a&gt;
+  &lt;a class="nav-element-link mt-05" href=""&gt;Grids&lt;/a&gt;
+&lt;/nav&gt;
+
```

Comment:
javaScript code missing for this component

Reply:
I have given a cdn link for script.js file